pytorch/pytorch_lightning_simple.py

```diff
@@ -53,27 +53,27 @@ def __init__(self, dropout: float, output_dims: List[int]):
 
         layers.append(nn.Linear(input_dim, CLASSES))
 
-        self.layers: nn.Module = nn.Sequential(*layers)
+        self.layers = nn.Sequential(*layers)
 
     def forward(self, data: torch.Tensor) -> torch.Tensor:
         logits = self.layers(data)
         return F.log_softmax(logits, dim=1)
 
 
 class LightningNet(pl.LightningModule):
-    def __init__(self, dropout: float, output_dims: List[int]):
+    def __init__(self, dropout: float, output_dims: List[int]) -> None:
         super().__init__()
         self.model = Net(dropout, output_dims)
 
     def forward(self, data: torch.Tensor) -> torch.Tensor:
         return self.model(data.view(-1, 28 * 28))
 
-    def training_step(self, batch, batch_idx: int) -> torch.Tensor:
+    def training_step(self, batch: torch.Tensor, batch_idx: int) -> torch.Tensor:
```

Comment:
nit 
```suggestion
    def training_step(self, batch: List[torch.Tensor], batch_idx: int) -> torch.Tensor:
```

Reply:
Thank you for your indication. I applied the changes.